src/components/data-exploration/heatmap/HeatmapPlot.jsx

```diff
@@ -45,9 +45,9 @@ const HeatmapPlot = (props) => {
 
   const dispatch = useDispatch();
 
-  const loadingGenes = useSelector((state) => state.genes.expression.loading);
+  const loadingGenes = useSelector((state) => state.genes.expression.downsampled.loading);
   const downsampledCellOrder = useSelector(
-    (state) => state.genes.expression.downsampledCellOrder,
+    (state) => state.genes.expression.downsampled.cellOrder,
```

Comment:
in `genes.expression.full` do we also have `cellOrder`?

Reply:
no, because the `cellOrder` is downsampling-specific: it tells you which cells are the result of the downsampling and in which order they are supposed to be shown